packages/webdriverio/src/middlewares.ts

```diff
@@ -36,9 +36,14 @@ export const elementErrorHandler = (fn: Function) => (commandName: string, comma
                 return result
             } catch (err: any) {
                 if (err.name === 'element not interactable') {
-                    const elementHTML = await element.getHTML()
-                    err.message = `Element ${elementHTML} not interactable`
-                    err.stack = err.stack ?? Error.captureStackTrace(err)
+                    try {
+                        return await element.waitForClickable()
+                    } catch (error) {
+                        const elementHTML = await element.getHTML()
+                        err.name = 'webdriverio(middleware): element did not become interactable'
+                        err.message = `Element ${elementHTML} did not become interactable`
```

Comment:
Plus point if you can build a script that is able to detect why the element is not interactable 😉